docs/mdfiles/simulcast.md

```diff
@@ -0,0 +1,71 @@
+OWT Simulcast Description
+---------------------
+
+# {#simulcast}
+The MCU server supports simulcast. This can be enabled through OWT Client SDK API. <br>
+
+1. Publish a simulcast stream
+~~~~~~{.js}
+//Example of simulcast publication
+conference = new Owt.Conference.ConferenceClient();
+// ...
+conference.join(token).then(resp => {
+    // ...
+    Owt.Base.MediaStreamFactory.createMediaStream(new Owt.Base.StreamConstraints(
+            audioConstraints, videoConstraints)).then(stream => {
+        /*
+         * Use `VideoEncodingParameters` as publish option.
```

Comment:
VideoEncodingParameters -> RTCRtpEncodingParameters. Please also provide a link to its definition, i.e.: https://w3c.github.io/webrtc-pc/#dom-rtcrtpencodingparameters.